v2/python/README.md

```diff
@@ -0,0 +1,54 @@
+# AdSense Management API v2 Samples
+
+A collection of command-line samples for the AdSense Management API.
+
+##Installation and first request
+
+1. Download Google APIs Client Library for Python (google-api-python-client) and
+   the OAuth library
+   (https://github.com/googleapis/google-auth-library-python-oauthlib):
+  - https://code.google.com/p/google-api-python-client/
+  - https://github.com/googleapis/google-auth-library-python-oauthlib
+
+  or use pip:
+
+  ```bash
+  $ pip install google-api-python-client
+  $ pip install google_auth_oauthlib
+  ```
+
+2. (No longer needed, check on this) Make sure you can import the client library:
+
+  ```
+  $ python
+  >>> import apiclient
+  ```
+
+3. Execute any of the scripts to begin the auth flow:
+
+  ```bash
+  $ python get_all_accounts.py
```

Comment:
```suggestion
  $ python3 get_all_accounts.py
```

Reply:
I actually think most people have mapped `python` to version 3.x at this point. Python 2 has been sunset for over a year. If you're fine with leaving this as is, can you resolve this conversation?